src/test/java/org/opentripplanner/netex/NetexBundleSmokeTest.java

```diff
@@ -157,9 +157,12 @@ private void assertTripPatterns(Collection<TripPattern> patterns) {
     TripPattern p = map.get(fId("RUT:JourneyPattern:12-1"));
     assertEquals("Jernbanetorget", p.getTripHeadsign());
     assertEquals("RB", p.getFeedId());
-    assertEquals("[<Stop RB:NSR:Quay:7203>, <Stop RB:NSR:Quay:8027>]", p.getStops().toString());
     assertEquals(
-      "[<Trip RB:RUT:ServiceJourney:12-101375-1000>]",
+      "[<Stop RB:NSR:Quay:7203 N/A>, <Stop RB:NSR:Quay:8027 N/A>]",
```

Comment:
Could I ask you to convert this to a Junit 5 test?

Reply:
This test is just touched by the PR, we will add and issue for converting all reminding JUnit <5.